include/gadget/sockets-map.h

```diff
@@ -199,16 +171,16 @@ gadget_socket_lookup(const struct __sk_buff *skb)
 	int off = l4_off;
 	switch (key.proto) {
 	case IPPROTO_TCP:
-		if (skb->pkt_type == PACKET_HOST)
-			off += offsetof(struct tcphdr, dest);
+		if (skb->pkt_type == SE_PACKET_HOST)
+			off += SE_TCPHDR_DEST_OFFSET;
 		else
-			off += offsetof(struct tcphdr, source);
+			off += SE_TCPHDR_SOURCE_OFFSET;
 		break;
 	case IPPROTO_UDP:
```

Comment:
Do we need namespaced versions of this and IPPROTO_TCP too?

Reply:
No, those are defined in an anonymous enum in vmlinux.h.